src/processors/BaseProcessor.h

```diff
@@ -161,5 +167,9 @@ class BaseProcessor : private JuceProcWrapper
     StringArray popupMenuParameterIDs;
     OwnedArray<ParameterAttachment> popupMenuParameterAttachments;
 
+    bool modulationPorts = false;
+    int inputModulationPort;
+    int outputModulationPort;
```

Comment:
Since these are uninitialized, other modules seem to show that they sometimes have modulation ports... anyway, this should be fixed with the other changes I had suggested, so probably nothing to worry about.